lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,20 +15,20 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="../../../assets/etcher.svg"
-            width="150px"
-            height="auto"
+          <svg-icon path="'../assets/etcher.svg'"
+            width="'150px'"
+            height="'auto'"
             os-open-external="https://etcher.io?ref=etcher_offline_banner"></svg-icon>
```

Comment:
In the other icon instances, you're using `os-open-external` in a `span` that wraps the icon, but here you're using it directly. What is the correct what of doing so? Can we always add it directly to the `svg-icon` element?